examples/webgl_loader_sea3d_morph.html

```diff
@@ -187,6 +182,9 @@
 
 				requestAnimationFrame( animate );
 
+				// Update SEA3D Animations
+				THREE.SEA3D.AnimationHandler.update( delta );
```

Comment:
I see `SEA3D.AnimationHandler` has been removed but don't see it being re-added anywhere, which doesn't necessarily have to mean it don't exist :), maybe it's just not so easy to find...

There are some more references, although most have been removed.